### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by spsquared** [spsquared/BattleBoxes_Multiplayer#41] [General]
Ask your questions about the game here! Don't spam please!

**Comment by SampleProvider** [comment]
WHY u DISABLE spam in GAM!!!!

**Reply by spsquared** [reply]
lol

**Comment by SampleProvider** [comment]
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
 how does this work
